### DISCUSSION TITLE
最新版本的蓝灯暂时不支持 Windows 7

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#194] [Announcements]
大家好，我们很遗憾地宣布，最新版本的蓝灯暂时不支持 Windows 7。对于给您带来的不便，我们深表歉意。我们已经意识到这些问题，并且正在解决这些问题，并且可以在即将发布的新版本中解决。请等待我们的更新，感谢您的耐心等待。

**Comment by woodchuckk** [comment]
啊？真是服了，果然跟猜测的一样啊……以后就纯维护吧，不要再拉着大家一起升级了本来线路就很不稳定，这边还有一大堆win7的用户啊

**Reply by MaxMKMax** [reply]
@woodchuckk 
很抱歉。我们正在开发新版本，很快就会发布。

**Reply by MaxMKMax** [reply]
已经尽力修复当前版本的问题。别担心。感谢您的等待。

**Comment by ycjdixon** [comment]
已经更新到7.8.1版本了，WIN7系统还是不能使用。请问什么时候修复。专业版付费用户急盼。

**Comment by MaxMKMax** [comment]
@woodchuckk 大家好! 想跟各位分享一个好消息! 很高兴告诉您我们已经解决了Window7的问题。这是您可以下载
蓝灯Window 7 的链接。
https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-win7.exe
